samples/MusicStore.Spa/Properties/launchSettings.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Doesn't VS auto generate one for you if it's missing? We've been ignoring it in Mvc - https://github.com/aspnet/Mvc/blob/dev/.gitignore#L37